get-extracts/get-extracts.py

```diff
@@ -1,146 +1,81 @@
-#!/usr/bin/env python
```

Comment:
What's the reason for removing this? Now that we support either version of Python, we should be safe to use whichever they have installed in their bin if invoked directly, right?